include/supercluster.hpp

```diff
@@ -187,7 +188,9 @@ class Supercluster {
 
             if (c.num_points == 1) {
                 const auto &original_feature = this->features[c.id];
-                result.emplace_back(point, original_feature.properties, original_feature.id);
+                // Generate feature id if options.generateId is set.
+                const auto featureId = options.generateId ? identifier{static_cast<std::uint64_t>(c.id)} : original_feature.id;
+                result.emplace_back(point, original_feature.properties, std::move(featureId));
```

Comment:
nit: `featureId` not movable as a constant.

Reply:
:) thanks, fixed